pimd/pim_vty.c

```diff
@@ -379,13 +379,19 @@ int pim_interface_config_write(struct vty *vty)
 							ij->group_addr,
 							group_str,
 							sizeof(group_str));
-						inet_ntop(AF_INET,
-							  &ij->source_addr,
-							  source_str,
-							  sizeof(source_str));
-						vty_out(vty,
-							" ip igmp join %s %s\n",
-							group_str, source_str);
+						if (ij->source_addr.s_addr == 0) {
```

Comment:
INADDR_ANY?

Reply:
@liam-mcb please compare against the standard macro `INADDR_ANY` when checking for a zero address

Reply:
Sorry, missed them in the first fix.